### DISCUSSION TITLE
Will mongoose@5.12.7 work with mpath@0.8.4 ?

### DISCUSSION THREAD

**Discussion by uzair004** [Automattic/mongoose#10676] [Q&A] [answered]
mongoose `5.12.7` uses mpath `v0.8.3 `which has some vulnerabilities, yes this issue was addressed in latest version of mongoose `v6.0`. here 3ee32b10c2ce086a0f0305749411a1268526d259 

## What I want to know

I am using mongoose `5.12.7` and want to use mpath `0.8.4 `with it instead of `0.8.3`, 

Should I upgrade mpath in older version of mongoose myself ?

will this break my code ?
will mongoose stop working ?

**Comment by vkarpov15** [comment, marked as answer]
We'll bump to mpath v0.8.4 in Mongoose v5.13.9. You should also be ok to manually upgrade to mpath@0.8.4 if necessary, but we recommend upgrading to v5.13.9 once it is released.

**Reply by uzair004** [reply]
Is there any breaking changes b/w mongoose `5.12.7` and `5.13.9` ?
I wanna update , but not sure If it will break my code.

**Reply by quick007** [reply]
*normally* non-major sem-ver updates don't make breaking changes to the API.
